petab/visualize/helper_functions.py

```diff
@@ -370,10 +370,26 @@ def check_ex_visu_columns(vis_spec, dataset_id_list, legend_dict):
     if X_SCALE not in vis_spec.columns:
         vis_spec.insert(loc=4, column=X_SCALE, value=LIN)
     if LEGEND_ENTRY not in vis_spec.columns:
-        if dataset_id_list is not [] and legend_dict is not {}:
+        # TODO:
+        # if we have dataset_id_list and legend_dict is empty
+        if dataset_id_list is not None and not bool(legend_dict):
```

Comment:
here we need to check that dataset_id_list is not empty, right? because [] is not None.

Reply:
hm... its: `dataset_id_list` is not None`, this should be correct, or?